aspnetcore/blazor/host-and-deploy/configure-linker.md

```diff
@@ -12,7 +12,7 @@ uid: blazor/host-and-deploy/configure-linker
 
 This article explains how to control the Intermediate Language (IL) Linker when building a Blazor app.
 
-Blazor WebAssembly performs [Intermediate Language (IL)](/dotnet/standard/managed-code#intermediate-language--execution) linking during a build to trim unnecessary IL from the app's output assemblies. The linker is disabled when building in Debug configuration. Apps must build in Release configuration to enable the linker. We recommend building in Release when deploying your Blazor WebAssembly apps. 
+Blazor WebAssembly performs [Intermediate Language (IL)](/dotnet/standard/glossary#il) linking during a build to trim unnecessary IL from the app's output assemblies. The linker is disabled when building in Debug configuration. Apps must build in Release configuration to enable the linker. We recommend building in Release when deploying your Blazor WebAssembly apps. 
```

Comment:
I think .NET has now standardized on "trimming" instead of "linking". I've opened https://github.com/dotnet/AspNetCore.Docs/issues/29089 to track switching the terminology.

Reply:
This is the old topic (3.1), which was replaced by the "trimmer" topic. I'll sweep the Blazor docs now to update to "trim"/"trimming."

Reply:
There wasn't much to change for late-model topics. All of our references to "linking"/"linker" look like they tie into the "Blazor linker" topic ...

https://learn.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/configure-linker?view=aspnetcore-3.1

... which is 3.1-only and was replaced at 5.0 by the "trimmer" topic ...

https://learn.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/configure-trimmer?view=aspnetcore-7.0

How do you want to proceed? ..... Drop all references to "linker"/"linking," including updating the 3.1 topic?

Reply:
> Drop all references to "linker"/"linking," including updating the 3.1 topic?

No need to mess around with the 3.1 topics. I care primarily about the docs for the supported versions: 6+.

Reply:
Ok ... in that case, I'll close your new issue. The last commit here resolves it.